README.md

```diff
@@ -8,30 +8,39 @@ get the following:
 - a run of a container based on this image
 
 All the logs will be in the container stdout/stderr, meaning you're able
-to rely on the container enging logging configuration in order to get a proper
+to rely on the container engine's logging configuration in order to get a proper
 status.
 
-### First run
-The script is based on a configuration file. It will be located in your current
-directory, for instance $PWD/.config/run_validations.conf.
+### Inventory file
+The repository holds an inventory file which holds connection information for
+all relevant hosts. You can add or edit host information here before the first
+run.
 
-This file will be generated once, unless you add the ```--regenerate``` option.
-Configuration file has priority, meaning you need to regenerate it if you want
-to modify some options. You can also edit it if you want.
+### First run
+The script first needs to build a container image which will be able to run the
+validations. To build the immage using the default options run:
+```Bash
+sudo ./validation.py --build
+```
+Please note that the default container cli is `podman` if you don't have podman
+installed use `--container=docker` instead.
 
-### Day+1 operation
-Once you have the configuration file you want, you can just call the script as
-follow:
+Once the container is built you can test it by running:
 ```Bash
-./validation.py --run
+sudo ./validation.py --run
```

Comment:
why "sudo" ? podman (and docker) can be launched as a user

Reply:
Because we get permission problems with the private key inside the container (when we switch users). OTOH I just made a test getting rid of the user creation entirely inside the container and just go with the root user. In which case I think we don't need sudo. 

IIRC we created the validation user to run validations inside the container (using --network host). But if connect from the container to the host via ssh we probably don't care about the user inside the container as long as we connect to the containerhost *as* the correct user.

Reply:
OK - try removing the non-root user. Since newer podman is out, we can run containers without the need of "sudo". This should help ensuring we still get some security since the "root" within the container will not match the host "root" when launched like that.

Reply:
Done